opentracing/harness/api_check.py

```diff
@@ -40,6 +40,14 @@ def check_baggage_values(self):
         """
         return True
 
+    def check_trace_identifiers(self):
+        """If true, the test will validate trace ID and :class:`Span` ID by
```

Comment:
Suggest to clarify what it means by "validating", i.e. that trace IDs match on parent and child, and span IDs are not the same. Not really sure what would be the example when this does not hold, aside from situations when a traced doesn't support the new methods altogether (i.e. returns blanks).

Reply:
> Not really sure what would be the example when this does not hold, aside from situations when a traced doesn't support the new methods altogether

I think this only stands for no-op `Tracer`, essentially ;)